### PR TITLE
Wildfly images renaming

### PR DESCRIPTION
**Pull request by jmesnil**

### CODE REVIEW

**Comment by jmesnil** [comment]
@jfdenise could you please review?

File: examples/https/helm.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 build:
   uri: https://github.com/wildfly/quickstart.git
-  ref: 26.1.0.Final
+  ref: 27.0.0.Final
   contextDir: helloworld-rs
   s2i:
     featurePacks:
       - org.wildfly:wildfly-galleon-pack:26.1.1.Final
```
Comment (jfdenise) [inline examples/https/helm.yaml:7]: Should be 27.0.0.Final

Comment (jmesnil) [reply examples/https/helm.yaml]: fixed, thanks

File: examples/todo-backend/todo-backend-s2i.yaml [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
   uri: https://github.com/wildfly/quickstart.git
-  ref: 26.0.0.Final
+  ref: 27.0.0.Final
   contextDir: todo-backend
   mode: s2i
   s2i:
     featurePacks:
-      - org.wildfly:wildfly-galleon-pack:26.1.1.Final
+      - org.wildfly:wildfly-galleon-pack:27.0.0.Final
+      - org.wildfly:wildfly-datasources-galleon-pack:2.0.2.Final
```
Comment (jfdenise) [inline examples/todo-backend/todo-backend-s2i.yaml:11]: Should be version 3.0.0.Final. Generally we order wildfly/cloud/datasources.

Comment (jmesnil) [reply examples/todo-backend/todo-backend-s2i.yaml:11]: fixed, thanks

**Review by jfdenise — APPROVED** [review]

**Pull request merged**